pkg/util/mount/fake.go

```diff
@@ -29,7 +29,6 @@ import (
 type FakeMounter struct {
```

Comment:
Fakes should go in a testing/ sub-pkg

Reply:
I don't disagree. Since that part of the code wasn't modified as part of this PR, I'd be happy to try and restructure that in a follow-on PR, most likely after this package is moved out of k/k.